contrib/kubernetes/kubernetes.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: loki-network
+  labels:
+    app: loki-network
+spec:
+  type: NodePort 
+  ports:
+    - protocol: UDP 
+      port: 1090 
+      name: udp-1
+      targetPort: 1090 
+    - protocol: UDP 
+      port: 1190 
+      name: udp-2
+      targetPort: 1190 
+  selector:
+    app: loki-network
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    description: loki-network lokinet
+    keel.sh/pollSchedule: "@every 1m"
+    keel.sh/notify: deployments # chat channels to sent notification to
+  generation: 1
+  labels:
+    app: loki-network
+    keel.sh/policy: force # update policy (available: patch, minor, major, all, force)
+    keel.sh/trigger: poll # enable active repository checking (webhooks and GCR would still work)
+    keel.sh/approvals: "0" # required approvals to update
+    keel.sh/match-tag: "true" # only makes a difference when used with 'force' policy, will only update if tag matches :dev->:dev, :prod->:prod
+  name: loki-network
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 3
+  revisionHistoryLimit: 20
+  selector:
+    matchLabels:
+      app: loki-network
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        description: loki-network lokinet
+      creationTimestamp: null
+      labels:
+        app: loki-network
+      name: loki-network
+    spec:
+      containers:
+        - image: 092763672147.dkr.ecr.us-east-1.amazonaws.com/loki/loki-network:latest
```

Comment:
Where is the dockerfile for this ?

Reply:
The docker file I used was docker/router.Dockerfile but the image should definitely be moved to docker hub in the future

Reply:
do we know if loki project has an official dockerhub?